README.md

```diff
@@ -78,6 +78,10 @@ If you want to propose a completely new feature, please create an RFC item. Good
 
 **Data API builder for Azure Databases** is licensed under the MIT license. See the [LICENSE](./LICENSE.txt) file for more details.
 
+## Third-Party Component Notice
+
+If you use the third-party software Banana Cake Pop by ChilliCream, Inc., as part of your Data API builder, you may have the option to save your work on ChilliCream's cloud service, via your separate ChilliCream account.  Microsoft is not affiliated with and does not endorse ChilliCream's service, and you use those services at your own option.
```

Comment:
should it be like `at your own risk` rather than `at your own option`

Reply:
That's what CELA wrote, I would leave it like they wrote :)